networkx/algorithms/connectivity/cuts.py

```diff
@@ -581,7 +581,7 @@ def minimum_edge_cut(G, s=None, t=None, flow_func=None):
 
         # Initial cutset is all edges of a node with minimum degree
         node = min(G, key=G.degree)
-        min_cut = G.edges(node)
+        min_cut = list(G.edges(node))
```

Comment:
I think both lines in this file should be ```set(G.edges(node))```.  At least that will make these always return a set. It's a bug not part of this PR, but this could be an easy way to fix it.